lib/gui/modules/drive-scanner.js

```diff
@@ -63,21 +64,7 @@ const emitter = new EventEmitter()
  */
 const BLOBS_DIRECTORY = path.join(__dirname, '..', '..', 'blobs')
 
-/**
- * @summary The usbboot "bootcode.bin" buffer
- * @type {Buffer}
- * @constant
- */
-const USBBOOT_BOOTCODE_BIN_BUFFER = fs.readFileSync(
-  path.join(BLOBS_DIRECTORY, 'usbboot', 'bootcode.bin'))
-
-/**
- * @summary The usbboot "start.elf" buffer
- * @type {Buffer}
- * @constant
- */
-const USBBOOT_START_ELF_BUFFER = fs.readFileSync(
-  path.join(BLOBS_DIRECTORY, 'usbboot', 'start.elf'))
+const BLOBS_CACHE = {}
```

Comment:
Not sure how I feel about the blobs being cached in a GUI module – any alternatives?

Reply:
Can you explain the concern? We could not cache them, which I think its fine anyways (the speed improvement of this particular caching case is probably minimal)

Reply:
Just as ref from the Flowdock thread; I'd move it into the usbboot adapter itself (as the UI shouldn't be concerned with config or internal state mgmt of something that's not the UI)